systemd/openqa-worker.target

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=openQA Worker
 Wants=var-lib-openqa-share.mount
+After=time-sync.target
```

Comment:
Hm, not sure if we should put this into `openqa-worker.target` or rather `openqa-worker@.service`?